libgnucash/backend/xml/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 # CMakeLists.txt for libgnucash/backend/xml
 
+add_subdirectory(DTD)
```

Comment:
@gjanssens You forgot to add DTD to the dist-list at the bottom. That's why the ubuntu CI failed: It runs distcheck instead of check.

Reply:
@jralls yea, I saw that and have the local fix already in my tree. I didn't push yet as there were some extra tweaks to do.